compiler/ast/src/ast_gen.rs

```diff
@@ -281,7 +281,7 @@ pub enum ExprKind<U = ()> {
 }
 pub type Expr<U = ()> = Located<ExprKind<U>, U>;
 
-#[derive(Debug, PartialEq)]
+#[derive(Copy, Clone, Debug, PartialEq)]
```

Comment:
```suggestion
#[derive(Debug, PartialEq)]
```
changed without code generation fix